group_vars/matrix_servers

```diff
@@ -1720,7 +1720,8 @@ matrix_synapse_account_threepid_delegates_msisdn: "{{ 'http://matrix-ma1sd:' + m
 matrix_synapse_container_client_api_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:8008' }}"
 #
 # For exposing the Matrix Federation API's plain port (plain HTTP) to the local host.
-matrix_synapse_container_federation_api_plain_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:8048' }}"
+matrix_synapse_container_default_federation_port: 8048
```

Comment:
This is the first definition of this variable. As such, it should be part of the `matrix-synapse` role, not in this hook-all-roles-together `group_vars/matrix_servers` file.

Relocating this to `roles/matrix-synapse/defaults/main.yml` is better. Renaming it to `matrix_synapse_container_federation_port` (without `_default`) is better.

Reply:
I agree, you are right with `matrix_synapse_container_federation_port`